Doc/whatsnew/3.15.rst

```diff
@@ -76,7 +76,7 @@ High frequency statistical sampling profiler
 --------------------------------------------
 
 A new statistical sampling profiler has been added to the :mod:`profile` module as
```

Comment:
I'm not sure it's the `profile` module now. Or is there a plan for having `profile.sample` that would import `profiling.sampling`?

@pablogsal

Reply:
The profile module is left the same and the only thing we need to do is mark it as deprecated. But that's pending work

Reply:
Indeed now is the profiling.sampling module 👍